bundle/language_en.properties

```diff
@@ -150,6 +152,8 @@ angal.admtype.editadmissiontype.title
 angal.agetype.agetypebrowser.title                                               = Age Type Browser
 angal.billbrowser.patientbillmanagment.title                                     = Patient Bills Management
 angal.dicomviewer.title                                                          = DICOM Viewer
+angal.dicomview.zoom.title                                                       = Zoon
```

Comment:
fix to "Zoom"

Reply:
Thanks; too many project; working too fast, and not enough sleep :(